package/batocera/core/batocera-configgen/configgen/configgen/generators/ecwolf/ecwolfGenerator.py

```diff
@@ -82,15 +81,15 @@ def generate(self, system, rom, playersControllers, metadata, guns, wheels, game
                 print(f"Error: couldn't go into directory {rom_path} ({e})")
 
         # File method .ecwolf (recommended) for command parameters, first argument is path to dataset,
-        # next parameters according ecwolf --help
+        # next parameters according ecwolf --help, use doppelquotes if path or filenames contains spaces
```

Comment:
`double quotes` not `doppelquotes`

Reply:
Oh no... What a slangmixup. This happens if everybody says: Wulfenstein srriidii 🤣